src/common/theme/typo.ts

```diff
@@ -0,0 +1,18 @@
+export const fontSizes = {
+  xs: '0.75rem',
+  sm: '0.875rem',
+  base: '1rem',
+  lg: '1.125rem',
+  xl: '1.25rem',
+  '2xl': '1.5rem',
+  '3xl': '1.875rem',
+  '4xl': '2.25rem',
+  '5xl': '3rem',
+  '6xl': '3.75rem',
+  '7xl': '4.5rem',
+  '8xl': '6rem',
+};
+
+export const fonts = {
```

Comment:
I think Archia font is using for heading and for body we can use Inter V:

![CleanShot 2023-01-05 at 3 30 16@2x](https://user-images.githubusercontent.com/53527664/210750229-6bda7609-2b53-42a2-90bc-764ef1d60a67.png)

![CleanShot 2023-01-05 at 3 30 56@2x](https://user-images.githubusercontent.com/53527664/210750274-b68a9ea6-c79f-4fd5-a6ae-036f97bf236b.png)

So we can do something like this:
```
import { RecursiveObject } from '@chakra-ui/react';

export const fonts: RecursiveObject<string> = {
  heading: `Archia, sans-serif`,
  body: `Inter V, Inter, system-ui, sans-serif`,
};

```

Reply:
yes, let me add Archia and use next/font